backend/graphql/resolvers/testResolvers.ts

```diff
@@ -131,6 +131,25 @@ const testResolvers = {
     ): Promise<string> => {
       return testService.deleteTest(id);
     },
+
+    publishTest: async (
+      _req: undefined,
+      { test_id }: { test_id: string },
+    ): Promise<TestResponseDTO | null> => {
+      const testToUpdate = await testService.getTestById(test_id);
+      if (!testToUpdate) {
+        throw new Error(`Test with id ${test_id} does not exist.`);
+      }
```

Comment:
```suggestion
```
this can be removed since an error will already be thrown by the service and `getTestById` is guaranteed to return a non-null object